src/utils/cta-utils-test.js

```diff
@@ -13,28 +13,124 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
+import {ActivityIframeView} from '../ui/activity-iframe-view';
```

Comment:
This is a lot of code so what do you think of moving it to something like `survey-utils.ts`?

Reply:
Sounds good!